Vagrantfile

```diff
@@ -1,19 +1,22 @@
-# -*- mode: ruby -*-
 # vi: set ft=ruby :
 
 Vagrant.configure("2") do |config|
-  config.vm.box = "ubuntu/zesty64"
-  config.vm.network "forwarded_port", guest: 4000, host: 4000, host_ip: "127.0.0.1"
-  config.ssh.forward_agent = true
+  config.vm.box = "pgrunwald/elixir-phoenix-ubuntu-trusty64"
+  config.vm.network "forwarded_port", guest: 4000, host: 1234, host_ip: "127.0.0.1"
 
-  config.vm.provision :shell, :inline => <<-EOT
-     echo LANG="en_US.utf8" > /etc/default/locale
-     echo LANGUAGE="en_US:" >> /etc/default/locale
-     echo LC_ALL="en_US.UTF-8" >> /etc/default/locale
-     sudo apt-get install python -y
-  EOT
+  config.vm.provision "shell", inline: <<-SHELL
+    apt-get update
+    apt-get update
+    apt-get install -y git
+    apt-get install -y elixir
 
-  config.vm.provision :ansible do |ansible|
-    ansible.playbook = "setup/playbook.yml"
-  end
+    cd /vagrant/
+    mix local.hex --force
+    mix local.rebar --force
+    mix deps.get
+    mix ecto.create
+    mix ecto.migrate
+    npm install --prefix ./assets
+    mix phx.server
```

Comment:
Yeah it's may be me being to lazy, it was a way to launch the app without login into the VM 😴